gugik_know_how.md

```diff
@@ -0,0 +1,89 @@
+# Instrukcja wyszukiwania i wyświetlania działek katastralnych na mapach Open Street Maps
+
+W tej instrukcji znajdziesz opisane sposoby, dzięki którym możesz pobrać dane działek katastralnych na terenie Polski i wyświetlić je (na przykład) na mapie [Open Street Map](https://www.openstreetmap.org/#map=6/52.018/19.137)] przy użyciu biblioteki [Leaflet](https://leafletjs.com/).
```

Comment:
Zdefiniujesz w nawiasie co to znaczy "działka katastralna"? Zgaduję, że nie tylko ja nie wiem.